src/components/HookForm/index.jsx

```diff
@@ -508,61 +538,76 @@ export default class HookForm extends Component {
               />
             </FormGroup>
           </ListItem>
+          <ListItem className={classes.displayBlock}>
+            <ListItemText
+              primary={
+                <Typography variant="subtitle1">
+                  Last Hook Fire Attempts
+                </Typography>
+              }
+            />
+            {hookLastFires && (
+              <DataTable
+                items={hookLastFires}
+                headers={['Task ID', 'FiredBy', 'Result', 'Attempted', 'Error']}
+                isPaginate
+                renderRow={hookFire => (
+                  <TableRow key={hookFire.taskId}>
+                    <TableCell>
+                      {(hookFire.result === 'SUCCESS' && (
+                        <Link to={`/tasks/${hookFire.taskId}`}>
+                          <Typography>{hookFire.taskId}</Typography>
+                        </Link>
```

Comment:
Right now it looks like:

![screen shot 2019-02-07 at 1 22 09 pm](https://user-images.githubusercontent.com/3766511/52433586-66a7fb00-2adb-11e9-9fd2-1e7f7822e4f9.png)

It should look like this:

![screen shot 2019-02-07 at 1 23 11 pm](https://user-images.githubusercontent.com/3766511/52433650-8ccd9b00-2adb-11e9-9df7-a5e272407a82.png)

This will make it look like any other link in a table to the rest of the app. Here's an example: [The Worker ID column in the Workers view](http://localhost:5080/provisioners/aws-provisioner-v1/worker-types/android-components-g).

Reply:
If the lastfire result is `success` then the taskId links to another route. But if result is `error` then no task of the mentioned `taskId` is actually created hence there is no link to the corresponding task. So I'm underlinking taskId for the sucess ones and no underline for `error` ones to distinguish them. Shall I change it ?

Reply:
From https://github.com/taskcluster/taskcluster-web/pull/413/files#r254810845, we could remove the link icon when the task has an error.